client/src/reviews/reviewParent.jsx

```diff
@@ -52,7 +52,9 @@ class ReviewParent extends React.PureComponent {
 
   // eslint-disable-next-line class-methods-use-this
   urlProductId() {
+    console.log('window.location', window.location.href);
     const questMarkLocation = (window.location.href).indexOf('?');
```

Comment:
If i understand what's going on here correctly. I think there's a more reliable way to do this using UrlSearchParams and window. location.search